.github/workflows/botonic-cli-tests.yml

```diff
@@ -1,6 +1,10 @@
 name: Botonic cli tests
 
-on: [push]
+on:
+  push:
+    paths:
+      - 'packages/botonic-cli/**'
+      - '.github/workflows/botonic-cli-tests.yml'
```

Comment:
are you sure that you need to include the workflow file itself?

Reply:
From what I understood from the [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths), a git diff is performed so I think it is needed.
I think I did a test on that but I will double check

Reply:
I did some checks on https://github.com/hubtype/botonic/tree/chore/codecov-temp branch and I confirm that line is needed: if we remove it, changes on the .yml file won't trigger the workflow.